modules/cms-kit/src/Volo.CmsKit.Common.Web/Renderers/MarkdownToHtmlRenderer.cs

```diff
@@ -33,10 +33,24 @@ public Task<string> RenderAsync(string rawMarkdown, bool allowHtmlTags = true, b
         {
             html = _htmlSanitizer.Sanitize(html);
         }
+        
+        if(referralLinks != null)
```

Comment:
Check for both null and whitespace situations:

```suggestion
        if(!string.IsNullOrWhiteSpace(referralLinks)
```